docs/TypeScript.md

```diff
@@ -229,10 +237,67 @@ In the last example we used interfaces to define the types for the request query
    ```
    Pay special attention to the imports at the top of this file. It might seem redundant, but you need to import both the schema files and the generated interfaces.
 
-Great work! Now you can make use of both JSON Schemas and TypeScript definitions. If you didn't know already, defining schemas for your Fastify routes can increase their throughput! Check out the [Validation and Serialization](Validation-and-Serialization.md) documentation for more info.
+Great work! Now you can make use of both JSON Schemas and TypeScript definitions.
+
+#### On-the-fly types
+
+If you do not want to generate types from your schemas, but want to use them diretly from your code, you can use the package
+[json-schema-to-ts](https://www.npmjs.com/package/json-schema-to-ts).
```

Comment:
I think we should add also alternatives.

Reply:
Do you know any?

What is your suggestion?

Reply:
The inverse order, like `typebox`

Reply:
Is there anything I can do?

Reply:
You can also introduce [`typebox`](https://www.npmjs.com/package/@sinclair/typebox) as described above.